lib/config/definitions.js

```diff
@@ -115,6 +115,45 @@ const options = [
     cli: false,
     env: false,
   },
+  // Yarn Lock Maintenance
+  {
+    name: 'maintainYarnLock',
+    description: 'Keep yarn.lock updated in base branch (no monorepo support)',
+    type: 'boolean',
+    default: false,
+  },
+  {
+    name: 'yarnMaintenanceBranchName',
+    description: 'Branch name template when maintaining yarn.lock',
+    type: 'string',
+    default: 'renovate/yarn-lock',
+    cli: false,
+    env: false,
+  },
+  {
+    name: 'yarnMaintenanceCommitMessage',
+    description: 'Commit message template when maintaining yarn.lock',
+    type: 'string',
+    default: 'Renovate yarn.lock file',
+    cli: false,
```

Comment:
I just realized this option is blocked on the command line.

Kept wondering why I couldn't specify it.

Would/could this be opened up so that it's available to be specified on the command line?